sdk/src/client/node_api/core/routes.rs

```diff
@@ -129,14 +130,38 @@ impl ClientInner {
         const PATH: &str = "api/core/v3/committee";
 
         let epoch_index = epoch_index.into().map(|i| format!("epochIndex={i}"));
-
         self.node_manager
             .read()
             .await
             .get_request(PATH, epoch_index.as_deref(), self.get_timeout().await, false, false)
             .await
     }
 
+    /// Returns information of all registered validators and if they are active.
+    /// GET JSON to /api/core/v3/validators
+    pub async fn get_validators(&self, page_size: Option<u32>) -> Result<ValidatorsResponse> {
+        const PATH: &str = "api/core/v3/validators";
+
+        let page_size = page_size.map(|i| format!("pageSize={i}"));
+        self.node_manager
+            .read()
+            .await
+            .get_request(PATH, page_size.as_deref(), self.get_timeout().await, false, false)
+            .await
+    }
+
+    /// Return the information of requested staker.
+    /// GET /api/core/v3/validators/{accountId}
+    pub async fn get_validator(&self, account_id: &AccountId) -> Result<AccountStakingResponse> {
```

Comment:
Something doesn't feel right to me here (I also mentioning this in the TIP draft):
*  The docs and the return type seem to suggest a name like: `get_staker` or `get_staker_info`
*  But the endpoint route seems to suggest that `get_validator` or `get_validator_info` is the more appropriate name, but then the docs need to change.

Reply:
This has been changed